code/modules/mob/living/carbon/carbon.dm

```diff
@@ -821,6 +821,8 @@
 		if(health <= HEALTH_THRESHOLD_DEAD && !HAS_TRAIT(src, TRAIT_NODEATH))
 			death()
 			return
+		if(HAS_TRAIT(src, TRAIT_DISSECTED))
```

Comment:
Any reason not to use HAS_TRAIT_FROM if we only care about removing it from that source?

Reply:
I think the autopsy bonus persisting but the dissection trait not is an important distinction. It means the coroner getting to people first is a lasting benefit.

Reply:
what I mean is that you only remove this trait if its source is `AUTOPSY_TRAIT` so you might as well use `HAS_TRAIT_FROM` instead of `HAS_TRAIT`, though currently there aren't any other sources of the trait anyway.

Reply:
ah okay I see what you mean